you can create now usesrs account lists / deleta account / add account

blayco opened the pull request.

blayco committed:
you can create now usesrs account lists / deleta account / add account

Konrad1221 commented on src/bank_group_work/Account.java:
```diff
@@ -5,7 +5,8 @@
     private String accountNumber;
     private AccountType accountType;
-    private BigDecimal balance;
+    private BigDecimal balance; //na bigdecimal nie da się normalnie dodawaći odejmować,
```
plus za dodawanie komentarzy

blayco replied:
As always :)

Konrad1221 replied:
dcfvfee

Konrad1221 commented on src/bank_group_work/Account.java:
```diff
@@ -5,7 +5,8 @@
     private String accountNumber;
     private AccountType accountType;
-    private BigDecimal balance;
+    private BigDecimal balance; //na bigdecimal nie da się normalnie dodawaći odejmować,
```
plus za komentarze

Konrad1221 commented on src/bank_group_work/Client.java:
```diff
@@ -1,12 +1,22 @@
 package bank_group_work;
 
+import java.lang.reflect.Array;
```
tu nie działa

blayco replied:
mi działa

blayco committed:
Thread 1.
  create new file
  add three lines of text

Thread 2.
  open  file
  print on screen content

blayco committed:
ThreadCollor Class added
